services/account/test/unit/test.ts

```diff
@@ -0,0 +1,127 @@
+// test/unit/test.js
+import {AccountController} from '../../controllers/AccountController';
```

Comment:
Indentation seems off. If you are using VSCode, you can use Prettier extension to automatically fix them for you. I also had this in my PR and Raymond suggested it :)